cert/aziot-certd/src/error.rs

```diff
@@ -22,6 +23,9 @@ impl std::fmt::Display for Error {
             Error::InvalidParameter(name, _) => {
                 write!(f, "parameter {:?} has an invalid value", name)
             }
+            Error::Unauthorized(user, id) => {
+                write!(f, "User {} not authorized to modify cert {}.", user, id)
```

Comment:
`"user {} is not authorized to modify the cert {}"`

(lower-case u, no period, grammar)

Reply:
Fixed